### ISSUE TITLE
AttributeError: 'ProjectSerializer' object has no attribute 'fields

### ISSUE THREAD

**Issue by moganty** [taigaio/taiga-back#824]
Not sure should I post here. I was getting below error when I browse api's. I setup the postgres and taiga on mac. Can some one give me hint what went wrong.

After setting up the postgres and taiga when I run in my browser 
Traceback (most recent call last):
  File "/Users/durga/Projects/taiga-back-master/tvenv/lib/python3.4/site-packages/django/core/handlers/base.py", line 174, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/durga/Projects/taiga-back-master/tvenv/lib/python3.4/site-packages/django/core/handlers/base.py", line 172, in get_response
    response = response.render()
  File "/Users/durga/Projects/taiga-back-master/tvenv/lib/python3.4/site-packages/django/template/response.py", line 160, in render
    self.content = self.rendered_content
  File "/Users/durga/Projects/taiga-back-master/taiga/base/response.py", line 100, in rendered_content
    ret = renderer.render(self.data, media_type, context)
  File "/Users/durga/Projects/taiga-back-master/taiga/base/api/renderers.py", line 633, in render
    context = self.get_context(data, accepted_media_type, renderer_context)
  File "/Users/durga/Projects/taiga-back-master/taiga/base/api/renderers.py", line 583, in get_context
    raw_data_post_form = self.get_raw_data_form(view, "POST", request)
  File "/Users/durga/Projects/taiga-back-master/taiga/base/api/renderers.py", line 524, in get_raw_data_form
    for name, field in serializer.fields.items():
AttributeError: 'ProjectSerializer' object has no attribute 'fields

**Comment by gangefors** [comment]
Did you figure out what was wrong with your setup?

I'm trying to get taiga-back v3.0.0 to work but I get the exact same error as you and I can't figure it out.

```
ERROR:2016-11-03 22:47:36,857: Internal Server Error: /api/v1/projects
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/site-packages/django/core/handlers/base.py", line 174, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python3.5/site-packages/django/core/handlers/base.py", line 172, in get_response
    response = response.render()
  File "/usr/local/lib/python3.5/site-packages/django/template/response.py", line 160, in render
    self.content = self.rendered_content
  File "/taiga/back/taiga/base/response.py", line 100, in rendered_content
    ret = renderer.render(self.data, media_type, context)
  File "/taiga/back/taiga/base/api/renderers.py", line 633, in render
    context = self.get_context(data, accepted_media_type, renderer_context)
  File "/taiga/back/taiga/base/api/renderers.py", line 583, in get_context
    raw_data_post_form = self.get_raw_data_form(view, "POST", request)
  File "/taiga/back/taiga/base/api/renderers.py", line 524, in get_raw_data_form
    for name, field in serializer.fields.items():
AttributeError: 'ProjectSerializer' object has no attribute 'fields'
```

**Comment by moganty** [comment]
Nope. You were right. HTML render was broken.

**Comment by bameda** [comment]
You need to make a pull in taiga-back, html renderer was delete a few days ago.

**Comment by gangefors** [comment]
I am using the released v3.0.0 for my Docker images. So that's why I see the error. 
I'll just have to wait for the next release to get that fixed. It's not a dealbreaker but I thought I had made an error during my install.
Thanks for the information.
